gapis/api/vulkan/api/image.api

```diff
@@ -210,6 +217,17 @@ cmd VkResult vkCreateImage(
       }
     }
   }
+
+  memRequirements := fetchImageMemoryRequirements(device, handle, hasSparseBit)
+  // If the vkCreateImage is inserted by GAPID (e.g. the staging image for
+  // reading framebuffer), NO memory requirements will be returned.
+  if memRequirements != null {
+    object.MemoryRequirements = memRequirements.MemoryRequirements
+    if (hasSparseBit) {
+      object.SparseMemoryRequirements = memRequirements.AspectBitsToSparseMemoryRequirements
```

Comment:
I'm not very sure about this 'map to map' assignment, is it valid/allowed in API files?

Reply:
It is allowed, but be aware maps are ref types. If this map is coming from an extra, you'll need to clone it so that modifications don't alter the original deserialized instance. See gles/extras.go for examples.